k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/quickstart/cwagent-fluent-bit-quickstart.yaml

```diff
@@ -148,6 +148,8 @@ spec:
             - name: devdisk
               mountPath: /dev/disk
               readOnly: true
+      nodeSelector:
+        kubernetes.io/os: linux
```

Comment:
These changes were unexpected. I'm not sure if they should be reverted.

Reply:
Should be fine. The update script dumps stuff from the adjacent yamls into this quickstart one. The change you see here is from #112